lldb/test/API/functionalities/target-new-solib-notifications/a.cpp

```diff
@@ -0,0 +1,7 @@
+extern "C" int b_function();
+
+int a_init() { return 234; }
+
+int a_global = a_init();

```

Comment:
Just delete all of this stuff, as it's not relevant for what we're testing. (the `a_function` -> `b_function` dep is marginally interesting, as it creates a more interesting dependency graph).

Reply:
Done.